pkg/storaged/crypto-keyslots.jsx

```diff
@@ -136,6 +136,7 @@ function clevis_unlock(block) {
 }
 
 export function unlock_with_type(client, block, passphrase, passphrase_type) {
+    console.log("UN", passphrase, passphrase_type);
```

Comment:
Debugging leftover, and a blocker as it logs passphrases.

Reply:
Ouch.